### ISSUE TITLE
Wrong execution before/after methods

### ISSUE THREAD

**Issue by voronsp** [saksmt/ktest#49]
Hello!
In case of addAfterEachHook and addBeforeEachHook methods Interceptor's constructure has wrong parameters. It must be vice versa:
```
fun SpecBuilder.addAfterEachHook(body: () -> Unit) {
        modify {
            currentSuite.addInterceptor(Interceptor(before = executeEveryTime(body)))
        }
    }

    fun SpecBuilder.addBeforeEachHook(body: () -> Unit) {
        modify {
            currentSuite.addInterceptor(Interceptor(after = executeEveryTime(body)))
        }
    }
```

**Comment by saksmt** [comment]
Thanks for your contribution again, will fix this by end of a day.

**Comment by saksmt** [comment]
Fixed in #50, released as RC3
